src/main/java/com/cloudbees/groovy/cps/tool/Driver.java

```diff
@@ -78,18 +79,21 @@ public void run() throws Exception {
              *      2. has a Closure parameter in one of the arguments, not in the receiver
              */
             Predicate<ExecutableElement> selector = (e) -> {
-                return e.getKind() == ElementKind.METHOD
-                    && e.getModifiers().containsAll(PUBLIC_STATIC)
-                    && e.getParameters().subList(1, e.getParameters().size()).stream()
-                        .anyMatch(p -> t.types.isAssignable(p.asType(), closureType))
-                    && !EXCLUSIONS.contains(e.getEnclosingElement().getSimpleName().toString()+"."+e.getSimpleName().toString());
+                boolean r = e.getKind() == ElementKind.METHOD
+                        && e.getModifiers().containsAll(PUBLIC_STATIC)
```

Comment:
So this won't work - there are a number of helper methods, like `protected static <T> T callClosureForMapEntry(Closure<T> closure, Map.Entry entry)`, that won't get translated but need to - see `each(Map,Closure)` calling `callClosureForMapEntry(Closure,Map.Entry)`, e.g.